hs_core/templates/search/indexes/hs_core/baseresource_text.txt

```diff
@@ -1,22 +1,33 @@
 {% load hydroshare_tags %} 
```

Comment:
Note that I removed `{% if %}` statements and replaced them with `|discoverable` filters to streamline processing.

Reply:
I've never noticed this file before... how is it used? I don't see any references to it, maybe it's used by some convention I'm not aware of?

Reply:
I see now it's part of haystack